setup.py

```diff
@@ -262,6 +262,7 @@ def run(self):
     entry_points={
         'console_scripts': [
             'borg = borg.archiver:main',
+            'borgmount = borg.archiver:main_mount',
```

Comment:
`mount.borg` would suggest that `mount -t borg` works, which doesn't.

With fuse and the PR in its current state it's either `borgmount $arch $mntpt` or `mount -t fuse.borgmount $arch $mntpt` (see `sshfs` for a good example).

So I'd rather suggest renaming the wrapper to `borgfs`.

I'm also going to add the second form of `mount -t` to the docs.